certified-connectors/Softools/apiDefinition.swagger.json

```diff
@@ -0,0 +1,879 @@
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.softools.net/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Intelligence;Data"
+    }
+  ],
+  "definitions": {
+    "Message": {
+      "type": "object",
+      "properties": {
+        "Message": {
+          "type": "string"
+        }
+      },
+      "example": {
+        "Message": "string"
+      }
+    },
+    "Records": {
+      "type": "object",
+      "example": "[\n  {\n    \"TextFieldExample\": \"\",\n    \"DateFieldExample\": null,\n    \"RangeFieldExample\": \"0\",\n    \"MultiStateHarveyBallFieldExample\": \"\",\n    \"MultiStateRAGFieldExample\": \"\",\n    \"BitFieldExample\": false,\n    \"NumberFieldExample\": 0,\n    \"SingleSelectionExample\": \"Item A\",\n    \"ListFieldExample\": {\n      \"changes\": { },\n      \"added\": [ ],\n      \"removed\": [ ]\n    }\n  },\n  {\n    \"TextFieldExample\": \"\",\n    \"DateFieldExample\": null,\n    \"RangeFieldExample\": \"0\",\n    \"MultiStateHarveyBallFieldExample\": \"\",\n    \"MultiStateRAGFieldExample\": \"\",\n    \"BitFieldExample\": false,\n    \"NumberFieldExample \": 0,\n    \"SingleSelectionExample\": \"Item A\",\n    \"ListFieldExample\": {\n      \"changes\": { },\n      \"added\": [ ],\n      \"removed\": [ ]\n    }\n  }\n]"
+    },
+    "basic-fields-response": {
+      "type": "object",
+      "properties": {
+        "identifier": {
+          "type": "string"
+        },
+        "Type": {
+          "type": "integer"
+        }
+      }
+    },
+    "List-record-response": {
+      "type": "array",
+      "items": {},
+      "example": [{
+        "_id": "id",
+        "QuickFilterSearchText": "",
+        "AccessRights": [
+          {
+            "_id": {
+              "$oid": "accessId"
+            },
+            "AccessType": 1,
+            "SourceType": 2,
+            "ActorType": 3,
+            "ActorId": "actorId",
+            "ParentId": null,
+            "Permissive": true
+          }
+        ],
+        "Id": "id",
+        "EditableAccessForUser": true,
+        "Tenant": "",
+        "IsArchived": false,
+        "TextFieldExample": "Example Text",
+        "DateFieldExample ": {
+          "$date": 1467331200000
+        },
+        "RangeFieldExample": "4",
+        "MultiStateHarveyBallFieldExample ": "75%",
+        "MultiStateRAGFieldExample ": "Amber",
+        "BitFieldExample ": true,
+        "NumberFieldExample": 20001,
+        "SingleSelectionExample": "Item A",
+        "ListFieldExample": {
+          "changes": {
+            "575fbf3cc99d15b138a08324": {
+              "TextField2Example": "Item 3"
+            }
+          },
+          "added": [
+            {
+              "Activity": "Item1",
+              "DateField2Example": {
+                "$date": 1466208000000
+              }
+            }
+          ],
+          "removed": [
+            "575fbf3cc99d15b138a08323"
```

Comment:
@thduttonuk Can you please remove secret-look-a-like values from any property as we may receive security alerts from Microsoft Security and we need to provide explanation?

Reply:
I will fix all the comments today. This isn't a key and just a random id as an example, I will remove.